source/2019-09-13-the-ember-times-issue-115.md

```diff
@@ -74,13 +75,13 @@ Be sure to check out [the full video](https://www.instagram.com/tv/B2CCM60AOZG/?
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Sinon QUnit 4.0 Released 🎉](https://twitter.com/JordanHawker/status/1170107247118053376)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Jordan Hawker (@elwayman02)](https://github.com/elwayman02) and [Steve Calvert (@scalvert)](https://github.com/scalvert) teamed up to bring us the best [Sinon.js](https://sinonjs.org/) experience possible. 💞
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Thanks to their work, [ember-sinon-qunit v4](https://github.com/elwayman02/ember-sinon-qunit) allows us to do a one-time setup, use the `sinon` object directly in our tests, and never worry about cleaning up mocks after the tests are run. To learn more about the motivation for change and the new syntax, you can check out [Jordan's blog post](https://www.jordanhawker.com/p/187541610821) and the [addon's README file](https://github.com/elwayman02/ember-sinon-qunit#usage).
+
+If you have been using Steve's [ember-sinon-sandbox](https://github.com/scalvert/ember-sinon-sandbox) or [ember-sinon-sinoff](https://github.com/scalvert/ember-sinon-sinoff), please check the [README file](https://github.com/elwayman02/ember-sinon-qunit#migrating-to-ember-sinon-qunit) to decide your migration path.
```

Comment:
I'm not exactly sure how this works in English, but would it be either

```suggestion
If you have been using Steve's [ember-sinon-sandbox](https://github.com/scalvert/ember-sinon-sandbox) or [ember-sinon-sinoff](https://github.com/scalvert/ember-sinon-sinoff), please check the [README file](https://github.com/elwayman02/ember-sinon-qunit#migrating-to-ember-sinon-qunit) to decide on your migration path.
```

or....

Reply:
Hmm, I'm not sure myself, haha. Maybe both?

Let's go with "choose" to be safe. :)